tests/renderer/setuptab.test.js

```diff
@@ -224,9 +227,13 @@ describe('Arguments form interactions', () => {
     spec.args.arg.type = 'csv';
     spec.args.arg.required = true;
     const {
-      findByText, findByLabelText, queryByText
+      findByText, findByLabelText, queryByText,
     } = renderSetupFromSpec(spec, UI_SPEC);
 
+    fetchValidation.mockResolvedValue(
+      [[Object.keys(spec.args), VALIDATION_MESSAGE]]
+    );

```

Comment:
I think this is the same mock setup as in `beforeEach`. Does it work if we remove this one?

Reply:
Yes it does!